src/sentry/issues/escalating.py

```diff
@@ -99,7 +99,7 @@ def query_groups_past_counts(groups: Sequence[Group]) -> List[GroupsCountRespons
     for g in groups:
         if g.issue_category == GroupCategory.ERROR:
             error_groups.append(g)
-        else:
+        elif g.issue_type.should_detect_escalation(g.organization):
```

Comment:
this condition should be sufficient since `ErrorGroupType` doesn't have `enable_escalation_detection = False`

Reply:
If the group category is not error, then we need to check if the non-error(performance included) issue type has escalation enabled. This prevents groups that have escalation disabled from being included in this query.

Reply:
Hm but the only GroupType that has `issue_category == GroupCategory.ERROR` is ErrorGroupType. It's fine as is too, but might be a good cleanup when we remove the flag so the branching is simplified.